Cargo.toml

```diff
@@ -10,6 +10,9 @@ strip = "debuginfo"
 codegen-units = 1
 panic = "abort"
 
+[features]
+salustsm = []
```

Comment:
If there are separate binary targets, do we still need a feature flag? If we want a feature flag + separate binary targets, is there a way to automatically set the feature based on which binary we're building?

Also some bikeshedding: I feel like "tsm_only" is a better name than "salustsm", as it's more obvious what the flag is doing (stripping out the non-TSM components).

Reply:
Feature flags can be enabled by default, but it does require the flag to be defined AFAIK. The nomenclature was arbitrary, so perhaps, we can just see what feels best in the channel.